samples/asyncapi/simple/sample.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Simple AsyncAPI
+summary: How to generate an AsyncAPI schema document from NServiceBus
+reviewed: 2025-08-13
+component: Core
+related:
+- samples/asyncapi/custom-message-types
+---
+
+This sample demonstrates how to generate an [AsyncAPI](https://www.asyncapi.com/en) schema document from NServiceBus endpoints using [Neuroglia.AsyncApi](https://github.com/asyncapi/net-sdk).
+
+> [!NOTE]
+> This is a sample only and should not be used in production as is.
```

Comment:
This is already implied by our samples documentation. Is there something else we want to convey or should we remove this part?

Reply:
As above - want to clarify it as it's not something we currently support. So how about:

```suggestion
> [!NOTE]
> AsyncAPI integration is not an officially supported feature of NServiceBus, and this example is for demonstration purposes only.
```